### PR TITLE
Dane/add/priorityandfairshare

### PR DESCRIPTION
**Pull request by kennedydane**
Added priority and fairshare information. Not sure if it's presented that well. Tried to just give a kind of summary with example commands so the users can check stuff for themselves. Also linked to the official slurm documentation...

### CODE REVIEW

File: docs/tech_docs/running_jobs.md [hunk 1]
```diff
@@ -237,3 +237,261 @@ The following table lists the parameters that can be used to describe the requir
...
 10. Email notifications can optionally be sent when a job's state changes. Create a comma-separated list with at least one of the following notification types: `NONE`; `BEGIN`; `END`; `FAIL`; `REQUEUE`; `ALL` (equivalent to `BEGIN,END,FAIL,REQUEUE,STAGE_OUT); STAGE_OUT`; `TIME_LIMIT`; `TIME_LIMIT_90` (reached 90 percent of time limit); `TIME_LIMIT_80` (reached 80 percent of time limit); `TIME_LIMIT_50` (reached 50 percent of time limit); and `ARRAY_TASKS`. ARRAY_TASKS will mean that a notification will be sent for each task in a job array (the default is only for the job array as a whole.)
 11. **Note that if you specify more memory or cores than is available on the nodes, or more nodes than are available on the cluster, _your job will never start but it will sit in the queue!_**
+
+## Understanding Priority and Fairshare
+The slurm documentation [on priority](https://slurm.schedmd.com/priority_multifactor.html) and [fairshare](https://slurm.schedmd.com/fair_tree.html) is very extensive, what follows here is a short summary of the principals behind priority and fairshare; and how to check what's going on with your jobs on ilifu.
```
Comment (Jordatious) [inline docs/tech_docs/running_jobs.md:242]: Can you swap the ; and , ?

The slurm documentation [on priority](https://slurm.schedmd.com/priority_multifactor.html) and [fairshare](https://slurm.schedmd.com/fair_tree.html) is very extensive; what follows here is a short summary of the principals behind priority and fairshare, and how to check what's going on with your jobs on ilifu.

**Comment by Jordatious** [comment]
I'd suggest this is a different page, perhaps linked from this page. We should make sure the other content is more easily accessible.

File: docs/tech_docs/running_jobs.md (hunk 1 above)
Comment (kennedydane) [reply docs/tech_docs/running_jobs.md]: Done

**Comment by kennedydane** [comment]
> I'd suggest this is a different page, perhaps linked from this page. We should make sure the other content is more easily accessible.

Happy to move it if someone has a suggestion as to where it should go.

**Comment by Jordatious** [comment]
Basically a new page under "tech docs"

**Comment by kennedydane** [comment]
@Jordatious and @jeremysmith123 

I've moved the things...

**Comment by Jordatious** [comment]
Sweet. Should we merge master in as well, or just let the PR take care of that? I guess the lack of conflicts means it's fine?

**Comment by kennedydane** [comment]
> Sweet. Should we merge master in as well, or just let the PR take care of that? I guess the lack of conflicts means it's fine?

I think the PR would've sorted it out, but I'll merged it in anyway.

**Comment by kennedydane** [comment]
@Jordatious or @jeremysmith123 — just wondering if we can merge this in?

**Pull request merged**